apps/web/app/(app)/environments/[environmentId]/components/PosthogIdentify.tsx

```diff
@@ -5,24 +5,28 @@ import { usePostHog } from "posthog-js/react";
 import { useEffect } from "react";
 
 import { env } from "@formbricks/lib/env.mjs";
+import { TTeam } from "@formbricks/types/teams";
 
 const posthogEnabled = env.NEXT_PUBLIC_POSTHOG_API_KEY && env.NEXT_PUBLIC_POSTHOG_API_HOST;
 
 export default function PosthogIdentify({
   session,
   environmentId,
+  team,
 }: {
   session: Session;
   environmentId: string;
+  team: TTeam;
```

Comment:
Ideally we would also pass `inAppSurveyBillingStatus`, `linkSurveyBillingStatus`, `userTargetingBillingStatus` here to not pass too many data to the client and save bandwidth.